packages/fx-core/templates/plugins/resource/bot/bicep/bot.template.bicep

```diff
@@ -0,0 +1,126 @@
+param botServiceName string
+param botServerfarmsName string
+param botWebAppSKU string = 'F1'
+param botServiceSKU string = 'F1'
+param botWebAppName string
+param botAadClientId string
+@secure()
+param botAadClientSecret string
+param botDisplayName string
+param authLoginUriSuffix string
+param m365ClientId string
+@secure()
+param m365ClientSecret string
+param m365TenantId string
+param m365OauthAuthorityHost string
+param m365ApplicationIdUri string
+{{#contains 'fx-resource-function' Plugins}}
+param functionEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-azure-sql' Plugins}}
+param sqlDatabaseName string
+param sqlEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-identity' Plugins}}
+param identityId string
+param identityName string
+{{/contains}}
+
+var botWebAppHostname = botWebApp.properties.hostNames[0]
+var botEndpoint = 'https://${botWebAppHostname}'
+var initiateLoginEndpoint = uri(botEndpoint, authLoginUriSuffix)
+
+resource botServices 'Microsoft.BotService/botServices@2021-03-01' = {
+  kind: 'azurebot'
+  location: 'global'
+  name: botServiceName
+  properties: {
+    displayName: botDisplayName
+    endpoint: uri(botEndpoint, '/api/messages')
+    msaAppId: botAadClientId
+  }
+  sku: {
+    name: botServiceSKU
+  }
+}
+
+resource botServicesMsTeamsChannel 'Microsoft.BotService/botServices/channels@2021-03-01' = {
+  parent: botServices
+  location: 'global'
+  name: 'MsTeamsChannel'
+  properties: {
+    channelName: 'MsTeamsChannel'
+  }
+}
+
+resource botServerfarm 'Microsoft.Web/serverfarms@2021-01-01' = {
+  kind: 'app'
+  location: resourceGroup().location
+  name: botServerfarmsName
+  properties: {
+    reserved: false
+  }
+  sku: {
+    name: botWebAppSKU
+  }
+}
+
+resource botWebApp 'Microsoft.Web/sites@2021-01-01' = {
+  kind: 'app'
+  location: resourceGroup().location
+  name: botWebAppName
+  properties: {
+    reserved: false
+    serverFarmId: botServerfarm.id
+    siteConfig: {
+      alwaysOn: false
+      http20Enabled: false
+      numberOfWorkers: 1
+    }
+  }
+  {{#contains 'fx-resource-identity' Plugins}}
+  identity: {
+    type: 'UserAssigned'
+    userAssignedIdentities: {
+      identityName: {}
```

Comment:
As we use the value of identityName for the key,  it should use the syntax like 
![image](https://user-images.githubusercontent.com/16380704/130409312-10c8a5f8-3fbb-4f69-8a26-cb0e4c244375.png)

Reply:
Fixed.